/voice_assistant_node:
  ros__parameters:
    snack_options: ['chips', 'chocolate', 'candies', 'nuts'] #TODO: handle chocolates or candy
    available_snacks_quantity: [5, 5, 5, 5]
    exit_cmd_options: ['bye', 'bubye', 'adios', 'ciao', 'no thanks', 'no thank you', 'see you', 'take care']
    speech_recognizer:
      energy_threshold: 200
      mic_device_index: 12
      phrase_time_limit: 4 # maximum number of seconds speech recognizer will allow a phrase to continue before stopping and returning the part of the phrase processed before the time limit was reached. The resulting audio will be the phrase cut off at the time limit. If ``phrase_timeout`` is ``None``, there will be no phrase time limit.
      timeout: 4 # maximum number of seconds speech recognizer will wait for a phrase to start before giving up. If ``timeout`` is ``None``, there will be no wait timeout.
      use_whisper: true
    text_to_speech_engine: elevenlabs # Options are: pyttsx3, elevenlabs
    use_sim_time: false
    